src/native/fs/fs_napi.cpp

```diff
@@ -763,6 +760,7 @@ struct FileWritev : public FSWrapWorker<FileWrap>
         auto buffers = info[1].As<Napi::Array>();
         const int buffers_len = buffers.Length();
         int total_len = 0;
+        _warn_threshold_ms = 400;
```

Comment:
is this really necessary to be hard coded here?
I would use the same value as the other ops unless it's not working.
If you still have to pass a different value - send it from js.